datafusion/substrait/tests/roundtrip_logical_plan.rs

```diff
@@ -373,4 +462,52 @@ mod tests {
             .await?;
         Ok(ctx)
     }
+
+    /// Cover all supported types
+    async fn create_all_type_context() -> Result<SessionContext> {
+        let ctx = SessionContext::new();
+        let mut explicit_options = CsvReadOptions::new();
+        let schema = Schema::new(vec![
+            Field::new("a", DataType::Boolean, true),
```

Comment:
I would personally find this test setup easier to use and understand if the fields were named for the type (for exmaple `"bool_col"` rather than "a"

The reason is that using just letters means that it is hard to understand what a particular test is testing without looking at this function